solr/core/src/java/org/apache/solr/handler/admin/SecurityConfHandler.java

```diff
@@ -303,11 +306,11 @@ public String toString() {
             }
           });
 
-          this.apis = ImmutableList.copyOf(apis);
+          syncedApis = ImmutableList.copyOf(syncedApis);
         }
       }
     }
-    return this.apis;
+    return syncedApis;
```

Comment:
This probably didn't need too change. If the prev comment is fixed, this will return the mutable version....